changelogs/v208-kajabi.1/disable-env-vars-during-build.md

```diff
@@ -0,0 +1,8 @@
+## Support disabling env vars at build time.
+
+Allow users to disable certain environment variables at build time without having to unset them on the application. For example, when dual-booting an application, it is required to add an environment variable like DEPENDENCIES_NEXT=1 on the application, but having that environment variable set during the build would cause errors or at least incorrect installation, but disabling it on the application would force the app to be rebooted and reverted.
+
+Example `.heroku/build-env-disable-list`
+```
+DEPENDENCIES_NEXT SILENCE_WARNINGS
+```
```

Comment:
Just musing, but this in particular strikes me as something Heroku would have a hard time upstreaming, since the use case is _so_ specific.

Although I understand what you wrote above, I'm not sure I fully understand what having `DEPENDENCIES_NEXT` in the build environment actually negatively impacts wrt bootboot, especially since env vars are not exported in buildpacks automatically (but I'm not sure what heroku/ruby is doing). We may need to hop on a call for that one.